extensions/binding/org.eclipse.smarthome.binding.bluetooth/src/main/java/org/eclipse/smarthome/binding/bluetooth/ble/GattService.java

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) 1997, 2015 by Huawei Technologies Co., Ltd. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.bluetooth.ble;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.eclipse.smarthome.binding.bluetooth.BluetoothBindingConstants;
+
+public enum GattService {
```

Comment:
Yes - I meant to do something else with this class. It was a dumping ground for bluetooth definitions when I first started and it hasn't moved... I think it should be included within the respective classes in io.transport...